__tests__/objects/paypal-button.spec.js

```diff
@@ -0,0 +1,31 @@
+// Libraries
+import React from 'react'
+
+// Object
+import PayPalButton from '../../src/objects/paypal-button'
+
+test('renders correctly', () => {
+  // Arrange
+  global.paypal = {
+    Buttons: {
+      driver: jest.fn(() => {
+        return jest.fn()
+      })
+    }
+  } // set mock window.paypal
+  const paypalCreateOrder = jest.fn()
+  const paypalOnApprove = jest.fn()
+  const onClick = jest.fn()
+
+  // Act
+  const wrapper = shallow(
+    <PayPalButton 
+      paypalCreateOrder={paypalCreateOrder}
+      paypalOnApprove={paypalOnApprove}
+      onClick={onClick}
+    />
+  )
+
+  // Assert
+  expect(wrapper).toMatchSnapshot()
```

Comment:
I'd add some additional assertions here as a sanity-check for what is being rendered. A snapshot test alone can get accidentally updated/broken by running jest with the `-u` flag.